config/config.go

```diff
@@ -22,22 +25,34 @@ type SafeConfig struct {
 	C *Config
 }
 
+func init() {
+	configReloadSuccess = prometheus.NewGauge(prometheus.GaugeOpts{
+		Name: "successful_config_load",
```

Comment:
This should have a `Namespace` set to `blackbox_exporter`.

Reply:
config_last_reload_successful is the suffix we use in Prometheus, which we should align with

Reply:
done and done